api/tests/certification/enrolment/unit/domain/usecases/get-user-certification-eligibility_test.js

```diff
@@ -601,24 +617,66 @@ describe('Certification | Enrolment | Unit | Usecases | get-user-certification-e
             }),
           );
           complementaryCertificationKey = 'NOT CLEA';
+          complementaryCertificationBadgeRepository.findAll.resolves([
+            domainBuilder.certification.enrolment.buildComplementaryCertificationBadge({
+              id: complementaryCertificationBadgeId,
+              requiredPixScore,
+            }),
+          ]);
         });
-        context('when acquired badge is outdated', function () {
-          context('when user has an acquired certification for this badge', function () {
-            it('should not be added in the eligibilities of the model', async function () {
-              // given
-              complementaryCertificationCourseRepository.findByUserId.withArgs({ userId }).resolves([
-                domainBuilder.certification.enrolment.buildComplementaryCertificationCourseWithResults({
+
+        context('when user has no pix certification', function () {
+          const isOutdated = false;
+          const offsetVersion = 0;
+          beforeEach(function () {
+            pixCertificationRepository.findByUserId.withArgs({ userId }).resolves([]);
+          });
+          it('should not be added in the eligibilities of the model', async function () {
+            complementaryCertificationCourseRepository.findByUserId.withArgs({ userId }).resolves([]);
+            certificationBadgesService.findLatestBadgeAcquisitions
+              .withArgs({
+                userId,
+                limitDate,
+              })
+              .resolves([
+                domainBuilder.buildCertifiableBadgeAcquisition({
                   complementaryCertificationBadgeId,
-                  results: [
-                    {
-                      source: sources.PIX,
-                      acquired: true,
-                      complementaryCertificationBadgeId,
-                    },
-                  ],
+                  complementaryCertificationKey,
+                  complementaryCertificationBadgeImageUrl: 'monImageUrl',
+                  complementaryCertificationBadgeLabel: 'monLabel',
+                  isOutdated,
+                  offsetVersion,
                 }),
               ]);
 
+            const userCertificationEligibility = await getUserCertificationEligibility(dependencies);
+
+            expect(userCertificationEligibility).to.deep.equal(
+              domainBuilder.certification.enrolment.buildUserCertificationEligibility({
+                id: userId,
+                isCertifiable,
+                certificationEligibilities: [],
+              }),
+            );
+          });
+        });
+        context(
+          'when user has a not cancelled not rejected validated pix certification without the required pixscore',
```

Comment:
`when user has a validated certification that is not cancelled not rejected and without the required pixscore `

Reply:
de toute façon plus j'y pense, plus ça parait overkill comme formulation, si ta certif est validated elle n'est ni cancelled ni rejected